### PR TITLE
feat(inventories-exporter): include createdAt and lastModifiedAt

### LINKED ISSUE #724: Include createdAt and lastModifiedAt in inventories export
### Support reference number

https://jira.commercetools.com/browse/SUPPORT-3718

### PR DESCRIPTION
**Pull request by daern91**
#### Support reference number
https://jira.commercetools.com/browse/SUPPORT-3718

#### Summary

PR adds `createdAt` and `lastModifiedAt` to inventories-exporter when doing CSV export.

#### Todo

- Tests
  - [x] Unit
  - [x] Integration

resolves #724

### CODE REVIEW

File: integration-tests/cli/inventories-exporter.it.js [hunk 1]
```diff
@@ -81,7 +81,8 @@ describe('StockExporter CLI', () => {
       )
       expect(stdout && stderr).toBeFalsy()
       const data = await fs.readFile(csvFilePath, { encoding: 'utf8' })
-      expect(data).toMatchSnapshot()
+      const expectedData = `12345,20,inventory-custom-type,integration tests!! arrgggh,${new Date().getFullYear()}`
```
Comment (adnasa) [inline integration-tests/cli/inventories-exporter.it.js]: ooh... the pain behind that "arrgggh"

File: packages/inventories-exporter/test/main.spec.js [hunk 2]
```diff
@@ -149,11 +148,7 @@ describe('InventoryExporter', () => {
           return Promise.resolve()
         })
       const outputStream = streamtest.v2.toText((error, result) => {
-        const expectedResult = stripIndent`
-          sku,quantityOnStock,restockableInDays
-          hello,me,4
-        `
-        expect(result).toEqual(expectedResult)
+        expect(result).toMatchSnapshot()
```
Comment (adnasa) [inline packages/inventories-exporter/test/main.spec.js]: why is this snapshotted, and tested differently from `inventories-exporter.it.js`?

File: integration-tests/cli/inventories-exporter.it.js (hunk 1 above)
Comment (daern91) [reply integration-tests/cli/inventories-exporter.it.js]: lol yeah... its been there for ages though and I dont wanna change it in this PR

File: packages/inventories-exporter/test/main.spec.js (hunk 2 above)
Comment (daern91) [reply packages/inventories-exporter/test/main.spec.js]: Good point, fixed in https://github.com/commercetools/nodejs/commit/88cf2beb30cc3dec84abec280fb895ae538ea590

**Comment by adnasa** [comment]
:shipit:

**Pull request merged**